src/hotspot/share/prims/jvmtiAgentList.cpp

```diff
@@ -243,6 +243,19 @@ bool JvmtiAgentList::is_dynamic_lib_loaded(void* os_lib) {
   }
   return false;
 }
+#ifdef AIX
+bool JvmtiAgentList::is_dynamic_lib_loaded(long device, long inode) {
+  JvmtiAgentList::Iterator it = JvmtiAgentList::agents();
+  while (it.has_next()) {
+    JvmtiAgent* const agent = it.next();
+    if (!agent->is_static_lib() && device && inode &&
```

Comment:
Style nit: we don't use implicit booleans so check `device != 0` and  `inode != 0` explicitly please.

Reply:
I followed your suggestion